src/winetricks

```diff
@@ -16162,7 +16162,10 @@ load_wmp9()
     w_override_app_dlls MPSetup.exe native pidgen
 
     w_try_cd "$W_CACHE"/"$W_PACKAGE"
-    w_try "$WINE" MPSetup.exe $W_UNATTENDED_SLASH_Q
+    w_try cabextract -d "$W_TMP" ./MPSetup.exe
```

Comment:
Please only change this for 64-bit (i.e., make a separate code path behind `if [ "$W_ARCH" = "win64" ]`.